frontend/views/containers/forms/FormPassword.vue

```diff
@@ -11,8 +11,8 @@ label.field
       :name='name'
       :data-test='name'
       v-model='$v.form[name].$model'
-      @input='e => $emit("input")'
-      @blur='e => $emit("blur")'
+      @input='debounceField("password")'
+      @blur='updateField("password")'
```

Comment:
Same question: did you check whether they need to even be here? :)

If not, they can be removed here, and elsewhere where they aren't needed (in the other files).

And if they are needed, an explanation of why `.$model` is insufficient